### PR TITLE
update 

### PR DESCRIPTION
**Pull request by shwetachan**
Fixes #2253 

#### Checklist:
- [ Y] 4 space indentation.
- [Y ] Coding conventions are followed.
- [ ] Input is taken dynamically.
- [Y ] Sample Input / Output is added at the end of file.
- [ Y] Logic Documentation (Comments).
- [ Y] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- javascript update
- C update

#### Languages Used:
- C
- JS in each file

#### Files Added:
- LIS_C.cpp
- LIS.js

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by asha15** [comment]
you was assigned to create only from JS . could you please remove the cpp file?

**Comment by SakshiShreya** [comment]
Also, complete the PR template

File: Longest_Increasing_Subsequence/LIS.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
+/**
+ * Dynamic programming approach to find longest increasing subsequence.
+ * Complexity: O(n * n)
+ *
+ * @param {number[]} sequence
+ * @return {number}
+ */
+export default function dpLongestIncreasingSubsequence(sequence) {
+  // Create array with longest increasing substrings length and
```
Comment (SakshiShreya) [inline Longest_Increasing_Subsequence/LIS.js:9]: Please use 4 spaces for indentation.

File: Longest_Increasing_Subsequence/LIS.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  let longestIncreasingLength = 0;
+
+  for (let i = 0; i < lengthsArray.length; i += 1) {
+    if (lengthsArray[i] > longestIncreasingLength) {
+      longestIncreasingLength = lengthsArray[i];
+    }
+  }
+
+  return longestIncreasingLength;
+}
```
Comment (SakshiShreya) [inline Longest_Increasing_Subsequence/LIS.js:53]: Add sample input and output at the EOF.

**Comment by somya-kapoor** [comment]
Are you working on it?

**Comment by somya-kapoor** [comment]
Due to no activity for more than 7 days, I am closing this issue.

**Pull request closed without merging**